libs/core/src/true-affected.ts

```diff
@@ -41,23 +55,29 @@ export const trueAffected = async ({
         }),
   });
 
-  const implicitDeps = (
-    projects.filter(
-      ({ implicitDependencies = [] }) => implicitDependencies.length > 0
-    ) as Required<TrueAffectedProject>[]
-  ).reduce(
-    (acc, { name, implicitDependencies }) =>
-      acc.set(name, implicitDependencies),
-    new Map<string, string[]>()
-  );
-
   projects.forEach(
-    ({ sourceRoot, tsConfig = join(sourceRoot, 'tsconfig.json') }) => {
+    ({ name, sourceRoot, tsConfig = join(sourceRoot, 'tsconfig.json') }) => {
       const tsConfigPath = resolve(cwd, tsConfig);
 
       if (existsSync(tsConfigPath)) {
+        if (verbose) {
+          logger.log(
```

Comment:
@EladBezalel maybe a change in approach, maybe we can start using `logger.debug` and then we can set the debug level or the DEBUG env var and this way u won't need to do the `if (verbose)` part and the logger should know how to handle log levels.

wdyt? the benefit here is that this will reduce the amount of branching in code and will make it a bit nicer and u won't have to add additional logic for params